testsuite/tests/typing-modules/Test.ml

```diff
@@ -137,10 +137,11 @@ Error: Signature mismatch:
          sig type t += E of int end
        is not included in
          sig type t += E end
-       Extension declarations do not match:
```

Comment:
This seems to be a regression.

Reply:
Indeed, typo on my part. I'll push a fix now.